ansible/environments.yml

```diff
@@ -103,3 +103,18 @@
           image_name: "d324da4b-b827-44fb-8331-bfad41a28aca"
       tags:
         - dev2-env
+
+    - name: Ensure aws_dev environment stack
+      cloudformation:
+        stack_name: "uat"
+        state: "present"
+        region: "us-west-2"
+        disable_rollback: true
+        template: "../aws/ae-environment.yml"
+      args:
+        template_parameters:
+          EnvName: "at-uat-uswest2"
+          ApiPort: 3013
+          NodeCount: 10
+          AmiId: "ami-34a9364c" # AMI  ubuntu 16.04 with master user + python added.
```

Comment:
Isn't it 16.04.3 ?

Reply:
it should, changed comment. But to will confirm when stack will get deployed.